src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/con-2061-clear-entries-recipienttemplate.sql

```diff
@@ -0,0 +1,18 @@
+BEGIN TRANSACTION
```

Comment:
This file should be named something like 'con-2061-transfer-recipient-template-to-recipients'

Reply:
done